types/src/multihash.rs

```diff
@@ -0,0 +1,145 @@
+use cid::CidGeneric;
+use multihash::Multihash;
+use nmt_rs::simple_merkle::tree::MerkleHash;
+use nmt_rs::NamespaceMerkleHasher;
+use tendermint::hash::SHA256_HASH_SIZE;
+
+use crate::nmt::{Namespace, NamespacedHash, NamespacedHashExt, NamespacedSha2Hasher, NS_SIZE};
+use crate::{Error, Result};
+
+pub const MULTIHASH_NMT_CODEC_CODE: u64 = 0x7700;
+pub const MULTIHASH_SHA256_NAMESPACE_FLAGGED_CODE: u64 = 0x7701;
+pub const MULTIHASH_SHA256_NAMESPACE_FLAGGED_SIZE: usize = 2 * NS_SIZE + SHA256_HASH_SIZE;
```

Comment:
those feel inconsistent with `sample` and `axis` which don't have `MULTIHASH_` prefixes

Reply:
true, renamed these to match convention better 👍